mmv1/third_party/terraform/tests/resource_bigtable_gc_policy_test.go

```diff
@@ -406,7 +406,7 @@ func testAccBigtableGCPolicyExists(t *testing.T, n string) resource.TestCheckFun
 		}
 
 		for _, i := range table.FamilyInfos {
-			if i.Name == rs.Primary.Attributes["column_family"] {
+			if i.Name == rs.Primary.Attributes["column_family"] && rs.Primary.ID != "" {
```

Comment:
I'm not sure this change does anything - if `rs.Primary.ID == ""` this func [returns an error](https://github.com/GoogleCloudPlatform/magic-modules/blob/fe2b62298c03a37bfd553bcd70902ed6c07b8a83/mmv1/third_party/terraform/tests/resource_bigtable_gc_policy_test.go#L392) - so we already know that `rs.Primary.ID != ""` if we've gotten here.

Since this change is to make `d.SetId(fi.GCPolicy)` get called in a specific case, could we instead use something like one of the [builtin check functions](https://www.terraform.io/plugin/sdkv2/testing/acceptance-tests/teststep#builtin-check-functions) to check the exact value of the id field?

Reply:
Sure, I will check the value of Primary.ID. It won't be very useful once we set the nested GC policy in JSON format in "gc_rules".

I am going to check the value in the same function instead of using a built-in function.

Done.